security/pkg/server/ca/authenticate/kube_jwt.go

```diff
@@ -71,7 +83,18 @@ func (a *KubeJWTAuthenticator) Authenticate(ctx context.Context) (*Caller, error
 	}
 	id, err := a.client.ValidateK8sJwt(targetJWT, a.jwtPolicy)
 	if err != nil {
-		return nil, fmt.Errorf("failed to validate the JWT: %v", err)
+		// try to validate using remote cluster
+		id, err = a.validateRemoteK8sJwt(targetJWT)
```

Comment:
@howardjohn Is there a way to know the csr is from a specific cluster? Then we can get rid of this trying all logic